pq-crypto/kyber_r3/kyber512r3_fq_avx2.S

```diff
@@ -0,0 +1,126 @@
+#include "kyber512r3_cdecl_avx2.h"
+
+/*****.include "fq.inc"*****/
```

Comment:
Can you please add a more descriptive comment here?

Reply:
It is updated.